cmd/dqa/feedback/github.go

```diff
@@ -260,8 +262,10 @@ func NewGitHubReport(site, etl, cycle, token string) *GithubReport {
 	tk := &oauth2.Token{
 		AccessToken: token,
 	}
+
+	context := oauth2.NoContext
```

Comment:
Per convention and to prevent shadowing the `context` package, the local variable should be called `ctx`. This applies to the `issues/github.go` file as well. Also `oauth2.NoContext` can be replaced with `context.Background()` since it is a no-op behind the scenes anyway.